app/merge_badges.py

```diff
@@ -14,8 +14,8 @@
 
 if subprocess.call(['which', 'python3']) != 0:
     raise PackageNotFoundError("Package python3 not found")
-if subprocess.call(['which', 'pdftk']) != 0:
-    raise PackageNotFoundError("Package pdftk not found")
+# if subprocess.call(['which', 'pdftk']) != 0:
```

Comment:
Why Commented? Please remove it

Reply:
It is because we are no longer using pdftk and it give assertion error on heroku because of the unmet dependencies. We are using `pyPDF2` in place of `pdftk`. @vaibhavsingh97

Reply:
Done :+1: